webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue

```diff
@@ -228,13 +282,20 @@ export default {
       newsFormTitlePlaceholder: `${this.$t('news.composer.placeholderTitleInput')}*`,
       newsFormSummaryPlaceholder: this.$t('news.composer.placeholderSummaryInput'),
       newsFormContentPlaceholder: `${this.$t('news.composer.placeholderContentInput')}*`,
+      items: [
+        {key: 'private',value: 'private', text: this.$t('news.composer.private'), align: 'center', sortable: false},
+        {key: 'shared',value: 'shared', text: this.$t('news.composer.shared'), align: 'center', sortable: false},
+      ],
       newsFormContentHeight: '250',
       newsFormSummaryHeight: '80',
       initDone: false,
       initIllustrationDone: false,
       showDraftNews: false,
       postingNews: false,
       savingDraft: false,
+      isCurentUserRedactor: false,
```

Comment:
No need to add an additional isCurentUserRedactorOrAuthor service,  this information can retrieved from this.currentSpace.isRedactor